exec/java-exec/src/main/java/org/apache/drill/exec/physical/base/PhysicalVisitor.java

```diff
@@ -55,10 +56,10 @@
 
 
   public RETURN visitExchange(Exchange exchange, EXTRA value) throws EXCEP;
+  public RETURN visitSingleMergeExchange(SingleMergeExchange exchange, EXTRA value) throws EXCEP;
```

Comment:
The same question as for `PrelVisitor.java`. Is it necessary to have separate `visitSingleMergeExchange`?